### PR TITLE
mungebit crashes otherwise

### PR DESCRIPTION
**Pull request by rylandely**

### CODE REVIEW

**Comment by rylandely** [comment]
This is a major issue with the lockfile system we are using.  I cannot even change my local version of syberiaMungebits and then use it right away because every R session or new stagerunner demands that I revert to the remote version.

**Comment by kirillseva** [comment]
@rylandely Mr. Grey will see you now

File: R/parse-datetime.r [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 #' @param date contains the date to be formatted
 #' @param mode gives the desired output format (see above)
-datetime_fn <- function(createdat, mode="since") {
+datetime_fn <- function(createdat, mode="since", permissive=FALSE) {
```
Comment (peterhurford) [inline R/parse-datetime.r]: Space dat `=`.

Comment (peterhurford) [inline R/parse-datetime.r]: Also, if you're going to add a new parameter, you should add roxygen documentation for it. :-1:

Comment (robertzk) [inline R/parse-datetime.r]: @rylandely can you follow up?

**Pull request merged**